recipes/xvfbwrapper/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.8" %}
+
+package:
+  name: xvfbwrapper
+  version: {{ version }}
+
+source:
+  fn: xvfbwrapper-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/x/xvfbwrapper/xvfbwrapper-{{ version }}.tar.gz
+  md5: c8ce7c35fc555a6af272e7e17cf0b5c0
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Weird. I guess I'm wrong about this one. Pretty sure it was in the [source code]( https://github.com/cgoldberg/xvfbwrapper/blob/1cd001b7f60d28297462d128ff97f36dcf8a9775/setup.py#L9-L12 ). Maybe it's not in a release yet. Well let's drop `setuptools` and these arguments. Sorry about that.